lms/templates/dashboard/_dashboard_certificate_information.html

```diff
@@ -18,103 +18,132 @@
-  % elif cert_status['status'] in ('generating', 'ready', 'notpassing', 'restricted', 'auditing', 'unverified'):
-    <p class="message-copy">${_("Your final grade:")}
-    <span class="grade-value">${"{0:.0f}%".format(float(cert_status['grade'])*100)}</span>.
-      % if cert_status['status'] == 'notpassing':
-        % if enrollment.mode != 'audit':
-          ${_("Grade required for a {cert_name_short}:").format(cert_name_short=cert_name_short)} <span class="grade-value">
-        % else:
-          ${_("Grade required to pass this course:")} <span class="grade-value">
-        % endif
-        ${"{0:.0f}%".format(float(course_overview.lowest_passing_grade)*100)}</span>.
-      % elif cert_status['status'] == 'restricted' and enrollment.mode == 'verified':
-      <p class="message-copy">
-        ${Text(_("Your verified {cert_name_long} is being held pending confirmation that the issuance of your {cert_name_short} is in compliance with strict U.S. embargoes on Iran, Cuba, Syria and Sudan. If you think our system has mistakenly identified you as being connected with one of those countries, please let us know by contacting {email}. If you would like a refund on your {cert_name_long}, please contact our billing address {billing_email}")).format(email=HTML('<a class="contact-link" href="mailto:{email}">{email}</a>.').format(email=settings.CONTACT_EMAIL), billing_email=HTML('<a class="contact-link" href="mailto:{email}">{email}</a>').format(email=settings.PAYMENT_SUPPORT_EMAIL), cert_name_short=cert_name_short, cert_name_long=cert_name_long)}
-      </p>
-      % elif cert_status['status'] == 'restricted':
-      <p class="message-copy">
-        ${Text(_("Your {cert_name_long} is being held pending confirmation that the issuance of your {cert_name_short} is in compliance with strict U.S. embargoes on Iran, Cuba, Syria and Sudan. If you think our system has mistakenly identified you as being connected with one of those countries, please let us know by contacting {email}.")).format(email=HTML('<a class="contact-link" href="mailto:{email}">{email}</a>.').format(email=settings.CONTACT_EMAIL), cert_name_short=cert_name_short, cert_name_long=cert_name_long)}
-      </p>
-      % elif cert_status['status'] == 'unverified':
-      <p class="message-copy">
-        ${Text(_("Your certificate was not issued because you do not have a current verified identity with {platform_name}. ")).format(platform_name=settings.PLATFORM_NAME)}
-        <a href="${reverify_link}"> ${Text(_("Verify your identity now."))}</a>
-      </p>
+    % if not cert_status['certificate_message_viewable']:
+      % if (cert_status['status'] == 'generating' or cert_status['status'] == 'downloadable'):
+        <p class="message-copy">
+          <%
+            certificate_available_date_string = course_overview.certificate_available_date.strftime('%Y-%m-%dT%H:%M:%S%z')
```

Comment:
This is broken on stage—my dashboard doesn't load. You should verify that `course_overview.certificate_available_date` is an actual `datetime` instance before calling `strftime`.

Reply:
FYI @edx/devops the release should be paused until this is fixed.

Reply:
I opened a PR for this here: https://github.com/edx/edx-platform/pull/16183